lib/userinterface.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module UserInterface
+  def welcome
+    puts 'Welcome to Tic Tac Toc Game!'
```

Comment:
I think you can make use of `main.rb` only and achieve the goal.
According to the instructions on pathwright, our `puts` and `gets.chomp` are to be inside the `main.rb`